dali/c_api/operator_trace_test.cc

```diff
@@ -27,31 +27,37 @@ using namespace dali;  // NOLINT
 
 struct OperatorTraceTestParam {
   int cpu_queue_depth, gpu_queue_depth;
-  bool exec_pipelined, exec_async;
+  bool exec_pipelined, exec_async, exec_dynamic;
 };
 
 namespace {
 
 OperatorTraceTestParam operator_trace_test_params_simple_executor[] = {
-        {1, 1, false, false},
-        {2, 2, false, false},
+        {1, 1, false, false, false},
+        {2, 2, false, false, false},
 };
 
 
 OperatorTraceTestParam operator_trace_test_params_pipelined_executor_uniform_queue[] = {
-        {2, 2, true, false},
-        {3, 3, true, false},
-        {2, 2, true, true},
-        {3, 3, true, true},
+        {2, 2, true, false, false},
+        {3, 3, true, false, false},
+        {2, 2, true, true, false},
+        {3, 3, true, true, false},
 };
 
 OperatorTraceTestParam operator_trace_test_params_pipelined_executor_separate_queue[] = {
-        {2, 3, true, true},
-        {3, 2, true, true},
-        {2, 3, true, false},
-        {2, 2, true, false},
+        {2, 3, true, true, false},
+        {3, 2, true, true, false},
+        {2, 3, true, false, false},
+        {2, 2, true, false, false},
 };
 
+OperatorTraceTestParam operator_trace_test_params_dynamic_executor[] = {
+  {1, 1, true, true, true},
+  {2, 2, true, true, true},
+};
```

Comment:
Looks like the formatting of this part is different from the similar part above

Reply:
Damn bug in VS code. https://github.com/microsoft/vscode/issues/241337